app/javascript/crayons/Links/__stories__/Links.stories.jsx

```diff
@@ -60,7 +60,7 @@ export const Block = (args) => <Link {...args} />;
 Block.args = {
   ...Inline.args,
   block: true,
-  variant: 'default',
+  variant: undefined,
```

Comment:
Refactor (blocking): If the variant prop is optional, omit the prop altogether. Or is it required for Storybook to be able to change the value with the Controls panel below the component in Storybook?

Reply:
It's optional, but it overwrites the one from `...Inline.args` (see 2 lines above), so it's more for this story presentation purpose.